pages/[slug].js

```diff
@@ -142,15 +146,30 @@ const Wizard = ({ children }) => {
   )
 }
 
-const Onboarding = () => (
-  <Wizard>
-    <Step1 />
-    <Step2 />
-    <Step3 />
-    <Step4 />
-    <Step5 />
-    <Step6 />
-    <Step7 />
-  </Wizard>
-)
+const Onboarding = ({ employer }) => {
+  return (
+    <Wizard employer={employer}>
+      <Step1 />
+      <Step2 />
+      <Step3 />
+      <Step4 />
+      <Step5 />
+      <Step6 />
+      <Step7 />
+    </Wizard>
+  )
+}
+
+Onboarding.getInitialProps = async ({ req }) => {
+  const slug = req.originalUrl.slice(1)
+  const res = await axios(
+    `http://localhost:3000/api/get-employer-from-slug?${slug}`
```

Comment:
we should update the url to get it from the window href or an env variable at some point